ibm_catalog.json

```diff
@@ -309,6 +310,40 @@
 								}
 							]
 						},
+						{
+							"key": "key_protect_plan",
+							"type": "string",
+							"default_value": "tiered-pricing",
+							"description": "The service plan of the Key Protect instance that will be provisioned by this solution. Only used if not supplying `existing_kms_instance_crn`. [Learn more](https://cloud.ibm.com/docs/key-protect?topic=key-protect-pricing-plan).",
+							"required": false,
+							"options": [
+								{
+									"displayname": "Standard",
+									"value": "tiered-pricing"
+								},
+								{
+									"displayname": "Cross-region Resiliency",
+									"value": "cross-region-resiliency"
```

Comment:
indentation seems odd here - did you use tabs instead of double spaces? Perhaps run a json pretty against the file